terragrunt/tools/grafana/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana
+
+This configuration is based on the latest published Grafana image, includes a CloudWatch data source, and provisions dashboards to enhance observability of SIRSI applications and infrastructure.
+
+```shell
+# In the following examples: 
+# - ave is alias for `aws-vault exec` command
+# - aws-switch-to-* is alias to set the:
+#   - AWS_PROFILE
+#   - TG_ENVIRONMENT
+#   - AWS_ENV (Not compulsory)
+#   - MFA_TOKEN handler (Out of scope of this documentation)
+# Feel free to use any convenient AWS profiler instead.
+```
+## build
+
+```shell
+docker build -t cdp-grafana .
```

Comment:
For consistency, can we use `cabinetoffice/cdp-grafana` tag pretty please?

Reply:
Like the consistency but I was not too keen about mixing an of-the-shelf tool with our services :thinking: 
Talking about consistency choosing a unified format for our env_vars makes me a very happy person :D 
![image](https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/assets/1422984/5aba55e3-2108-4299-b67a-772d5d6fa4f2)

Reply:
That's true. That's because these are just app config options and it's possible to override them by constructing env var name to correspond the config option.

What would you prefer them being named? We could give them a prefix by putting all app settings under a single key (i.e. CDP).

Also, note that these are case insensitive, so you could CAPITALISE them.

Reply:
hmm, being able to capitalise them all makes the maintenance of the infra side a touch easier. 
what about underscores? can we agree on naming like PERSON_SERVICE, instead of PERSONSERVICE.
also what about the number of the underscores? is there a rule like if it is .NET related (Framework Config) they must be double and if us defining (APP Configs) them they must be single?
Perhaps something for team chat instead of here but the current convention make my eyes bleed :D

Reply:
>   what about the number of the underscores?

That is something we cannot change. Double underscore maps to a nested configuration option in .net.

Reply:
> but the current convention make my eyes bleed 

There's no convention. It's accidental.